NOTICE

```diff
@@ -31,7 +28,7 @@ Pygments - Python syntax highlighter 2.12.0 : BSD 3-clause "New" or "Revised" Li
 Pyparsing 3.0.9 : MIT License
 pytest 7.0.1 : MIT License
 python-attrs 21.4.0 : MIT License
-python-certifi 2022.6.15 (https://github.com/certifi/python-certifi) : Mozilla Public License 2.0
+python-certifi 2022.6.15 (https://github.com/certifi/python-certifi) : Mozilla Public License 2.0
```

Comment:
(What changed on this line?)

Reply:
Same as other:

> The generated NOTICE file adds a character displaying as '^M' at the end of each line. Maybe it's a new line. I manually copied this line here since we made manual changes to the original file here because the MPL licence requires some kind of attribution. I will probably copy it in the future as well so I think it's good to solve it once and for all with this MR. The process will always be the same.